content/v2/services.markdown

```diff
@@ -0,0 +1,88 @@
+---
+title: Services | DNSimple API v2
+excerpt: This page documents the DNSimple services API v2.
+---
+
+# Services API
+
+* TOC
+{:toc}
+
+Services provide a way to apply many records to a domain at once.
+Additionally any changes made to a service will automatically be reflected in the domain's records.
```

Comment:
This specific line is incorrect, as at the time being we don't propagate the changes when a service is updated. I'd remove it. In fact, we may probably want to remove both lines for now as we haven't really described any of the Product in any API page so far.